server/forge/bitbucketdatacenter/bitbucketdatacenter.go

```diff
@@ -96,7 +96,7 @@ func (c *client) URL() string {
 func (c *client) Login(ctx context.Context, req *forge_types.OAuthRequest) (*model.User, string, error) {
 	config := c.newOAuth2Config()
 
-	// TODO: Add proper state and pkce...
+	// TODO: Add proper state and pkce... // cspell:disable-line // TODO: what is "pkce?!?"
```

Comment:
pkce is an extension of oauth

Reply:
ah thanks - It would be nice to add a link to some docu pages then :D 

and I'll add it as word